lib/workers/repository/process/write.ts

```diff
@@ -35,7 +42,24 @@ export async function writeUpdates(
   for (const branch of branches) {
     addMeta({ branch: branch.branchName });
     const branchExisted = branchExists(branch.branchName);
-    const res = await processBranch(branch);
+    let branchCache = cachedBranches?.find(
+      (br) => br.branchName === branch.branchName
+    );
+
+    if (!branchCache) {
+      logger.debug(`No branch cache found for ${branch.branchName}`);
```

Comment:
We should not log this unnecessarily if repo caching is off. It might just confuse/concern people. So we need to distinguish between these two uses cases:
- No cache in use: do not log
- Cache in use but no cache for this branch: debug log